docs/source/index.rst

```diff
@@ -123,29 +123,36 @@ If you use AiiDA for your research, please cite the following work:
 Acknowledgements
 ****************
 
-This work is supported by:
+AiiDA is supported or has been supported by:
 
 * the `MARVEL National Centre of Competence in Research`_ funded by the `Swiss National Science Foundation`_;
-* the `MaX European Centre of Excellence`_ funded by the Horizon 2020 EINFRA-5 program, Grant No. 676598 (2015-2018) and the Horizon 2020 INFRAEDI-2018-1 program, Grant No. 824143 (2018-2021);
-* the `swissuniversities P-5 project "Materials Cloud"`_.
+* the `MaX European Centre of Excellence`_ funded by the Horizon 2020 EINFRA-5 program, Grant No. 676598 and the Horizon 2020 INFRAEDI-2018-1 program, Grant No. 824143;
+* the `MarketPlace`_ project, funded by the Horizon 2020 H2020-NMBP-25-2017 call, Grant No. 760173;
+* the `INTERSECT`_ project, funded by the Horizon 2020 H2020-NMBP-TO-IND-2018 call, Grant No. 814487;
+* the `PASC`_ Swiss platform for Advanced Scientific Computing;
+* the `swissuniversities P-5 project "Materials Cloud"`_, project ID 182-008;
+* the `NFFA`_ project, funded by the Horizon 2020 programm, Grant No. 654360;
+* the BIG-MAP Battery 2030+ initiative;
+* `EPFL`_ (École Polytechnique Fédérale de Lausanne);
+* `Harvard University`_;
+* the `Bosch Research and Technology Center`_ (USA).
+
+A full list of supporting partners and institutions, with the respective logos, is available on the `AiiDA acknowledgements page`_.
```

Comment:
Since we are already linking to the acknowledgements page on the homepage, should we maybe just list all funding sources their and restrict it here to only the most important ones? Those lists are likely going to get out of sync and then we avoid the need to maintain both of them.

Reply:
Indeed, this was one of my comments/questions. I agree, and I have now simplified.